app/styles/preprint-detail-page.scss

```diff
@@ -0,0 +1,18 @@
+.popover {
+    max-width: 100%;
+}
+
+.popover-button {
+    margin-left: 7px;
+    height: 34px;
+}
+
+.popover-content {
+    background: white;
+    overflow: auto;
+
+    .form-group {
+        margin-bottom: 0px;
```

Comment:
px shouldn't be needed for 0

Reply:
I have to force the `form-group`'s bottom margin to be 0px because the default it inherited from somewhere else is not, and the default breaks the widget layout.

Reply:
ah gotcha